_data-prepper/pipelines/configuration/sources/otel-logs-source.md

```diff
@@ -0,0 +1,68 @@
+| proto_reflection_service | No | `false` | A boolean that enables a reflection service for Protobuf services (see [ProtoReflectionService](https://grpc.github.io/grpc-java/javadoc/io/grpc/protobuf/services/ProtoReflectionService.html) and [gRPC reflection](https://github.com/grpc/grpc-java/blob/master/documentation/server-reflection-tutorial.md) documents). |
+| unframed_requests | No |  | A `boolean` that enables requests which are not framed using the gRPC wire protocol. |
+| thread_count(Optional) | No | `500` | The number of threads to keep in the `ScheduledThreadPool`. |
+| max_connection_count | No | `500` | The maximum number of open connections allowed. |
+
+### SSL
+
+You can configure SSL in the `otel_logs_source` source with the following options.
+
+| Option | Required | Default | Description |
+| :--- | :--- | :--- | :--- |
+| ssl | No | `true` | A boolean that enables TLS/SSL. |
+| sslKeyCertChainFile | Yes, if `ssl` is set to `true`. | A `string` that represents the SSL certificate chain file path or AWS S3 path. S3 path example `s3://<bucketName>/<path>`. |
+| sslKeyFile | Yes, if `ssl` is set to `true`. | A `string` that represents the SSL key file path or AWS S3 path. S3 path example `s3://<bucketName>/<path>`. |
+| useAcmCertForSSL | No | `false` | A boolean enables TLS/SSL using certificate and private key from AWS Certificate Manager (ACM). |
+| acmCertificateArn | No | Yes, if `useAcmCertForSSL` is set to `true`. | A `string` that represents the ACM certificate ARN. ACM certificate take preference over S3 or local file system certificate. |
+| awsRegion | Yes, if `useAcmCertForSSL` is set to `true` or `sslKeyCertChainFile` and `sslKeyFile` is `AWS S3 path` |       | A `string` that represents the AWS region to use ACM or S3. Required. |
+
+## Usage
+
+To get started, create a `pipeline.yaml` file. See the following `YAML` file configuration example:
+
+```
+source:
+    - otel_logs_source:
+```
+
+## Metrics
+
+You can use the following metrics with the `otel_logs_source` source.
+
+| Option | Type | Description |
+| :--- | :--- | :--- | 
+| `requestTimeouts` | Counter | Measures the total number of requests that time out. | 
+| `requestsReceived` | Counter | Measures the total number of requests received by Otel logs source. |
+| `badRequests` | Counter | Measures the total number of requests that could not be parsed. |
+| `requestsTooLarge` | Counter | Measures the total number of requests that exceed the maximum allowed size. Indicates the size of the data to be written into the buffer is beyond buffer's maximum capacity. |
+| `internalServerError` | Counter | Measures the total number of requests that are erroneous due to any other reason than requestTimeouts or requestsTooLarge error. |
+| `successRequests` | Counter | Measures the total number of requests successfully written to the buffer. |
+| `payloadSize` | Summary | Measures the distribution of all incoming payload sizes. |
```

Comment:
Is `Summary` the correct category for `successRequests`?

Reply:
Let's stick with "Distribution summary"